packages/components/src/Typeahead/Typeahead.scss

```diff
@@ -36,7 +37,7 @@ $tc-typeahead-items-zindex: 1035 !default;
 
 	.items-container,
 	.items-body {
-		max-height: 70vh;
+		max-height: 32vh;
```

Comment:
This seems to be 3.2rem in classic mode and 4.8rem with category in the guideline. hmm this doesn't feel right, we should have a single size. Can you check with the UX ?

Reply:
This is about the items container, not the item itself. The usecase you're talking about is managed here : https://github.com/Talend/ui/pull/3114/files#diff-f73bbee543e731629b9cc27277a87ea17ccc17624cebe6e1f287e8f23a334a6bR191